com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/CopybookServiceImpl.java

```diff
@@ -236,7 +236,7 @@ public void observerCallback(RequiredCopybookEvent event) {
     path =
         findCopybook(
             requiredCopybookName,
-            (String) configurationSettingsStorable.getProfiles(),
+            configurationSettingsStorable.getProfiles().toString(),
```

Comment:
I've experienced that calling the `JsonObject#toString` this way will return a string with doubled quotes - `""string""`

Reply:
> I haven't seen any unit test that verify this functionality, we have to do that.

taking care of

Reply:
Is still pending this modification?

Reply:
modify it, I forgot about it, thanks

Reply:
I read about this and toString should be enough because we already have a Java Object and the JSON is parsed in the first instance